### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by svitalis123**
# Vet clinic database: create animals table
## I accomplished

- [ ] Install PostgreSQL
- [ ] Run it on SQL Shell
- [ ] Created Database
- [ ] Created Table
- [ ] Inserted Values into table columns
- [ ] Queried into the table 

Here is the screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,159 @@
...
+## 🛠 Built With <a name="built-with">PostgreSQL</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Loom video <a name="live-demo"></a>
+
+> A video Explaining The app
+- [Loom Video](https://www.loom.com/share/9a2411e8409d42c89e45e458f1a4ce84)
```
Comment (FrederickMih) [inline README.md:76]: - All sections that don't apply to this project, you are supposed to get rid of it. Please don't just copy/paste without going through the entire README template. This is not professional.

File: README.md [hunk 2]
```diff
@@ -1,48 +1,159 @@
...
+
+If you like this project create an issue with your comments.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for inspiring the design.
```
Comment (FrederickMih) [inline README.md:137]: - This line of code doesn't match your project.

File: README.md [hunk 3]
```diff
@@ -1,48 +1,159 @@
...
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+This project is [MIT](./LICENSE) licensed.
 
-## 📝 License
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This project is [MIT](./MIT.md) licensed.
```
Comment (FrederickMih) [inline README.md:48]: - This link is not working. Let's fix it to render our readme file professional :+1:

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: @svitalis123  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Descriptive PR :heavy_check_mark: 
- Correct SQL commands have been placed in the requested files :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by svitalis123** [comment]
Thank you i have implemented optional changes

**Pull request merged**